### PR TITLE
BE-2834: Added Increment version doc for Android

### PR DESCRIPTION
**Pull request by ftmzhrsckldr**

### CODE REVIEW

**Review by boztopuz — COMMENTED** [review]
@csonuryilmaz I reviewed this doc, I fixed some typos and grammer mistakes, all links are good, but we have already this documentation under Versioning. If this doc is necessary, it approved by me. 

https://docs.appcircle.io/versioning/android-version

File: docs/integrations/android-specific-workflow-steps/increment-build-and-version-number.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ If you are not incrementing the versioning process via **Appcircle** and have no
 :::
 
 ### Prerequisites
-The workflow steps that need to be executed before running the **Android Increment Build and Version Number** workflow step, along with their respective reasons, are listed in the table below:
+The workflow steps that need to be executed before running the **Increment Build and Version Number** workflow step, along with their respective reasons, are listed in the table below:
```
Comment (ftmzhrsckldr) [inline docs/integrations/android-specific-workflow-steps/increment-build-and-version-number.md:20]: @boztopuz The component name is: `Android Increment Build and Version Number` for that it must be with `Android`

File: docs/integrations/android-specific-workflow-steps/increment-build-and-version-number.md [hunk 2]
```diff
@@ -57,9 +57,9 @@ The outputs that can result from the operation of this component are listed as f
 
 | Variable Name                  | Description                                              |
 |--------------------------------|----------------------------------------------------------|
-| `$AC_ANDROID_NEW_VERSION_CODE` | Incremented version code that is written to the project. |
-| `$AC_ANDROID_NEW_VERSION_NAME` | Incremented version name that is written to the project. |
+| `$AC_ANDROID_NEW_VERSION_CODE` | Incremented version code is written to the project. |
+| `$AC_ANDROID_NEW_VERSION_NAME` | Incremented version name is written to the project. |
```
Comment (ftmzhrsckldr) [inline docs/integrations/android-specific-workflow-steps/increment-build-and-version-number.md:61]: @boztopuz Due to this change, the sentence has turned into a **sentence expressing a situation**, but I think that the 2nd column should be the **definition sentence** of the 1st column.

File: docs/integrations/android-specific-workflow-steps/increment-build-and-version-number.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+This component increments the `versionCode` and `versionName` according to the provided strategies.
+
+For more information about Android versioning, you can refer to this document: [Android Versioning](https://docs.appcircle.io/versioning/android-version/).
+
+:::caution
+If you are not incrementing the versioning process via **Appcircle** and have not enabled this feature through **Configuration**, this step completes the step without making any version changes.
+:::
+
+### Prerequisites
+The workflow steps that need to be executed before running the **Android Increment Build and Version Number** workflow step, along with their respective reasons, are listed in the table below:
```
[the remark this answers is not in the document]
Comment (ftmzhrsckldr) [reply docs/integrations/android-specific-workflow-steps/increment-build-and-version-number.md]: @boztopuz No is not common component, it in Android and iOS:
- https://github.com/appcircleio/appcircle-ios-build-version-increment
- https://github.com/appcircleio/appcircle-android-build-version-increment-component

[the remark this answers is not in the document]
Comment (boztopuz) [reply docs/integrations/android-specific-workflow-steps/increment-build-and-version-number.md]: Yes I know, the repo names can be different. But, in the UI the component names are same. Due to this, I changed it.

[the remark this answers is not in the document]
Comment (ftmzhrsckldr) [reply docs/integrations/android-specific-workflow-steps/increment-build-and-version-number.md]: @boztopuz Actually those are different repos and they have different input, output and source code. For that each component must have a specific doc page.

**Review by boztopuz — APPROVED** [review]

File: docs/integrations/android-specific-workflow-steps/increment-build-and-version-number.md (hunk 1 above)
Comment (csonuryilmaz) [reply docs/integrations/android-specific-workflow-steps/increment-build-and-version-number.md]: IMO @boztopuz missed the comment here, I will handle it. :raising_hand_man:

**Review by csonuryilmaz — APPROVED** [review]

**Pull request merged**